src/core/a-animation.js

```diff
@@ -463,3 +479,35 @@ function strToBool (str) {
 function boolToNum (bool) {
   return bool ? 1 : 0;
 }
+
+/**
+ * Converts a number 0-255 to hex
+ * @param {color}
+ * @returns {string}
+ */
+function componentToHex (color) {
+  var hex = color.toString(16);
```

Comment:
This is also possibly unneeded as the value that is passed comes from a THREE.Color which is already tested to be within 0 and 1 from a hex value correct?